addons/mail/static/src/core/web/chatter.js

```diff
@@ -194,11 +194,17 @@ export class Chatter extends Component {
                 } else {
                     this.state.showAttachmentLoading = false;
                     this.state.isAttachmentBoxOpened =
-                        this.props.isAttachmentBoxVisibleInitially && this.attachments.length > 0;
+                        this.attachments.length > 0 &&
+                        (this.props.isAttachmentBoxVisibleInitially ||
+                            this.state.isAttachmentBoxOpened);
```

Comment:
`isAttachmentBoxOpened` is `isAttachmentBoxVisibleInitially`. So, you can just check for `isAttachmentBoxOpened`.

Reply:
Yeah right